specs/language/expressions.tex

```diff
@@ -219,11 +222,14 @@
 \texttt{T[N]}.
 
 \p On expiration of the cxvalue, the value is assigned back to the argument
-lvalue expression following an inverted conversion if applicable. The argument
-expression must be of a type or able to convert to a type that has defined
-copy-initialization to and from the parameter type. The lifetime of the cxvalue
-begins at argument expression evaluation, and ends after the function returns. A
-cxvalue argument is passed by-address to the caller.
+lvalue expression using a resolved assignment expression as if the expression
+\texttt{Arg = Param} were written\footnote{The argument expression is not
```

Comment:
Is a space character needed between 'written' and '\footnote' here?

Reply:
Without a space the footnote mark is right next to the end of the word, and it is still correctly parsed.
<img width="1087" alt="Screenshot 2024-09-04 at 6 59 10 PM" src="https://github.com/user-attachments/assets/ecbea9a8-3a2a-4b32-bf68-225e0d067bfb">